### DISCUSSION TITLE
How to get SecGen working on proxmox

### DISCUSSION THREAD

**Discussion by schemesec** [cliffe/SecGen#335] [General]
I've tried for days with multiple installs and configurations and I just haven't been able to get SecGen working. I'm not finding really any tutorials on getting this setup. SecGen sounds like an amazing thing, but I haven't been able to use it.

**Comment by cliffe** [comment]
Hi! Sorry to hear that you are finding it difficult to get started. We use SecGen daily with Proxmox, but you have to use the vagrant proxmox plugin fork from my repository. Also, you will need to create the base boxes, following the guidance in README. We are currently mostly using Debian Bookworm with KDE and Kali Linux.

Good luck! If you have technical details of where you are going stuck feel free to share.

**Reply by schemesec** [reply]
That's good! I was afraid that it was potentially broken. I've got the vbox route(running under proxmox) and the proxmox route with proxmox api and have encountered a lot of issues involving dependencies. I'll probably give it a fresh go sometime soon or maybe buy another pc to run it. I have tried to install vagrant-proxmox using your source but I'm having trouble figuring out how to use it properly. I'm likely just having trouble figuring out how to install it because im not all that familiar with vagrant and i'm not sure where in the file system i'm supposed to go to make these changes. I did have two questions for now, is it possible to get vagrant to ignore cert issues? I haven't added a signed cert to my proxmox install and I think its an issue. 
`ConnectProxmox: SSL_connect returned=1 errno=0 peeraddr=192.168.1.100:8006 state=error: certificate verify failed (unable to get local issuer certificate)`
I was also wondering, if I need to be calling ruby secgen.rb run any differently if I want to use proxmox to provision the vm?
